### PR TITLE
Robosuite bugfix

### PR DESCRIPTION
**Pull request by Tanmay-Kulkarni101**
Fix bugs required to run robosuite with primitives.

### CODE REVIEW

File: experiments/robosuite/dreamer/dreamer_v2_single_task_primitives_lift.py [hunk 1]
```diff
@@ -101,7 +103,7 @@
                 use_dm_backend=True,
                 max_path_length=5,
             ),
-            image_kwargs=dict(),
+            # image_kwargs=dict(),
```
Comment (mihdalal) [inline experiments/robosuite/dreamer/dreamer_v2_single_task_primitives_lift.py:106]: delete

File: experiments/robosuite/dreamer/dreamer_v2_single_task_primitives_lift.py [hunk 2]
```diff
@@ -117,7 +119,7 @@
             model_hidden_size=400,
             stochastic_state_size=50,
             deterministic_state_size=200,
-            embedding_size=1024,
+            # embedding_size=1024,
```
Comment (mihdalal) [inline experiments/robosuite/dreamer/dreamer_v2_single_task_primitives_lift.py:122]: delete

File: rlkit/envs/wrappers/primitives_wrappers.py [hunk 3]
```diff
@@ -746,6 +746,7 @@ def reset(self):
             .transpose(2, 0, 1)
             .flatten()
         )
+
```
Comment (mihdalal) [inline rlkit/envs/wrappers/primitives_wrappers.py:749]: delete

File: experiments/robosuite/dreamer/dreamer_v2_single_task_primitives_lift.py (hunk 1 above)
Comment (Tanmay-Kulkarni101) [reply experiments/robosuite/dreamer/dreamer_v2_single_task_primitives_lift.py:106]: done

File: experiments/robosuite/dreamer/dreamer_v2_single_task_primitives_lift.py (hunk 2 above)
Comment (Tanmay-Kulkarni101) [reply experiments/robosuite/dreamer/dreamer_v2_single_task_primitives_lift.py:122]: done

File: rlkit/envs/wrappers/primitives_wrappers.py (hunk 3 above)
Comment (Tanmay-Kulkarni101) [reply rlkit/envs/wrappers/primitives_wrappers.py:749]: done.

**Pull request merged**